src/server/routes/api.js

```diff
@@ -169,6 +169,21 @@ module.exports = function(app) {
         .then(() => res.json({success : true}));
     });
 
+  api.get('/admin/columns', requireAuth, roleAuth(roles.ROLE_ADMIN),
+    (_, res) => {
+      let columns = Object.entries(User.schema.paths)
+        .reduce((acc, [key, value]) => {
+          if (!('displayName' in value.options)) {
+            return acc;
+          }
+
```

Comment:
We should have something here to add GPA, majorGPA, food, classRequirement only when their specific options are set.